src/java.base/share/classes/java/lang/Object.java

```diff
@@ -246,12 +253,14 @@ public boolean equals(Object obj) {
      * the unsigned hexadecimal representation of the hash code of the
      * object. In other words, this method returns a string equal to the
      * value of:
-     * <blockquote>
-     * <pre>
+     * {@snippet lang=java :
      * getClass().getName() + '@' + Integer.toHexString(hashCode())
-     * </pre></blockquote>
-     *
-     * @return  a string representation of the object.
+     * }
+     * The {@link java.util.Objects#toIdentityString(Object)
+     * Objects.toIdentityString} method returns the string for an
```

Comment:
Separately, is it just me or this sentence could be rephrased to not read like `toIdentityString(Object)` accepts an object equal to the string (that would ...)?